ontology/occo.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: ontology_detail
+id: occo
+title: Occupation Ontology
+browsers:
+- title: BioPortal Ontology Browser
+  label: BioPortal
+  url: https://bioportal.bioontology.org/ontologies/OCCO?p=classes
+contact:
+  email: zhengj2007@gmail.com
+  github: zhengj2007
+  label: Jie Zheng
+  orcid: 0000-0002-2999-0103
+description: The Occupation Ontology (OccO) is an ontology in the domain of human occupations. OccO is an initial ontological presentation of information taken from a standardized occupation taxonomy, the US Standard Occupational Classification (SOC) as enhanced by the O*Net system.
```

Comment:
A better description would be good:
- rather than saying "in the domain of human occupations" say what exactly the ontology is about - the domain is not that important; as a user I want to know what kind of terms i will find, and how I will use the ontology.
- `initial ontological presentation` -> this sounds like its not supposed to be used yet ('initial'). Also, `presentation` should probably be representation.
- What is the O*Net system? (maybe say something in brackets"
- Say something clear about how you want OCCO to be used.

Reply:
Updated the description per request.

Reply:
Thank you!